src/App.tsx

```diff
@@ -34,17 +35,24 @@ function App() {
   return (
     <div class='absolute inset-0 flex items-center justify-center border-2 border-red-500'>
       <div class='flex flex-col items-center justify-center gap-4'>
-        <button onClick={() => setCount(count() + 1)}>Click me if you dare.</button>
-        <button onClick={addNode}>Add Node</button>
+        <button onClick={handleClick}>Click me if you dare.</button>
         <p class={`text-[10rem] font-bold ${
```

Comment:
ok better to factor out the logic and use `clsx` to simplify class construction.
particularly in UX the more you can factor out the logic, the simpler and less error prone the code will be.

```ts
type Fizzy = { fizz?: boolean, buzz?: boolean, quiz?: boolean }

const getFizzy = (n: number): Fizzy => ({
  fizz: n % 3,
  buzz: n % 5,
  quiz: n % 7,
})

...

const { fizz, buzz, quiz } = getFizzy(count());
class={clsx(fizz && 'text-green-500', buzz && 'text-orange-500', quiz && 'text-purple-500')}

```